<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#dialog{
background-color:rgb(245, 245, 245);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Yu Mincho Light&quot;;color:rgb(0, 0, 0);
font: 25 48pt &quot;Yu Mincho Light&quot;;
font: 25 28pt &quot;Yu Mincho Light&quot;;
font: 25 18pt &quot;Yu Mincho Light&quot;;
font: 25 24pt &quot;Yu Mincho Light&quot;;
font: 25 24pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string>Learn &amp; Protect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="learn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 25 12pt &quot;Yu Mincho Light&quot;;
background-color:rgb(0, 0, 0);color:rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>LEARN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>20</width>
     <height>341</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="activate">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 25 12pt &quot;Yu Mincho Light&quot;;
background-color:rgb(0, 0, 0);color:rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>ACTIVATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deactivate">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 25 12pt &quot;Yu Mincho Light&quot;;
background-color:rgb(0, 0, 0);color:rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>DEACTIVATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Yu Mincho Light&quot;;color:rgb(0, 0, 0);
font: 25 18pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string>Learn Pattern</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Yu Mincho Light&quot;;color:rgb(0, 0, 0);
font: 25 18pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string>Protect System</string>
   </property>
  </widget>
  <widget class="QLabel" name="condition">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Yu Mincho Light&quot;;color:rgb(255, 255, 255);
font: 25 18pt &quot;Yu Mincho Light&quot;;
font: 7pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="learningprogress1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Yu Mincho Light&quot;;color:rgb(0, 0, 0);
font: 25 14pt &quot;Yu Mincho Light&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="learningprogress2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Yu Mincho Light&quot;;color:rgb(255, 255, 255);
font: 25 14pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Yu Mincho Light&quot;;color:rgb(0, 0, 0);
font: 25 14pt &quot;Yu Mincho Light&quot;;
font: 25 18pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
